operations/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package operations defines the entry points for the application and CLI.
```

Comment:
Maybe I don't know what "entry points" refers to here, but can rephrase it as something like "commands for the Evergreen CLI, which serves are the binary for both the Evergreen server and CLI". Is it worth mentioning that the application server and client are one and the same CLI?